index.html

```diff
@@ -16,17 +16,18 @@
                 <a href="#"><img class="logo" alt="Logo icon" src="img\Logo Placeholder.svg"></a>
                 <nav class="menu">
                     <ul>
-                        <li>
-                            <p>Portfolio</p>
+                        <li class="nav-item">
+                            <a href="#work"><p>Portfolio</p></a>
                         </li>
-                        <li>
-                            <p>About</p>
+                        <li class="nav-item">
+                            <a href="#aboutContainer"><p>About</p></a>
                         </li>
-                        <li>
-                            <p>Contact</p>
+                        <li class="nav-item">
+                            <a href="#contactForm"><p>Contact</p></a>
                         </li>
                     </ul>
-                    <img class="bar" src="img\Icon - Menu.svg" alt="hambuger icon">
+                    <img class="bar open-menu" src="img\Icon - Menu.svg" alt="hambuger icon">
+                    <img class="bar close-menu" src="img\Icon - Cancel.svg" alt="hambuger icon">
                 </nav>
             </header>
             <section class="content">
```

Comment:
[ Required ] The animation on the `h1` tag and the `p` tag appear to be causing the menu items on the mobile navigation menu to overlap with the contents of the page. please see the screenshot below for more details on this. You can fix this by removing the `animation-fill-mode: forwards; ` CSS rule from your animation styling rules.

```CSS
.content h1 {
  animation-name: title-intro;
  animation-duration: 2s;
}

.content p {
  animation-name: paragraph-intro;
  animation-duration: 2s;
}
```

<img src="https://user-images.githubusercontent.com/16392046/201783163-e9590674-4f87-43e5-b398-eae9ae4ddb88.png" width="300px" />